spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurationsReporterConfigurationTests.java

```diff
@@ -56,10 +57,11 @@ void shouldNotSupplyReporterIfSenderIsMissing() {
 
 	@Test
 	void shouldBackOffOnCustomBeans() {
-		this.contextRunner.withUserConfiguration(CustomConfiguration.class).run((context) -> {
-			assertThat(context).hasBean("customReporter");
-			assertThat(context).hasSingleBean(Reporter.class);
-		});
+		this.contextRunner.withUserConfiguration(SenderConfiguration.class, CustomConfiguration.class)
```

Comment:
It seems that the test code was not correct because it did not include `SenderConfiguration.class`.